packages/docusaurus-plugin-google-analytics/src/analytics.ts

```diff
@@ -32,18 +38,19 @@ export default (function() {
   }
 
   /* eslint-disable */
-  (function(i, s, o, g, r, a, m) {
```

Comment:
Can we avoid touching this function? This is straight out of Google Analytics and embed code and we're better off not modifying it if possible.

Reply:
Actually i also wonder if its worth to convert this to typescript. I still prefer UI stuff to be in JS to avoid extra compile step. And to sort of demonstrate that you can write plugin in js.

For important stuff like docs plugin, its important to have typing because the functionality is sort of complex enough.

Reply:
Ok!
This change is absolutely safe.
But, I agree with this plugin not necessary to refactor to Typescript.